content/en/docs/reference/setup-tools/kubefed/kubefed-join.md

```diff
@@ -94,6 +94,6 @@ kubefed join CLUSTER_NAME --host-cluster-context=HOST_CONTEXT [flags]
 ```
 
 ### SEE ALSO
-* [kubefed](kubefed.md)	 - kubefed controls a Kubernetes Cluster Federation
+* [kubefed](/docs/reference/setup-tools/kubefed/kubefed/)	 - kubefed controls a Kubernetes Cluster Federation
```

Comment:
Making changes to auto-generated files but those files don't get a fresh date a few lines below? This same comment for similar files below and above.